CMSgov/easi-app
# [EASI 3840] data page

# EASI-3840


## Changes and Description

- Add currently available fields to data page based on spike EASI-3640
- Add mock data for CEDAR exchanges



## How to test this change

- Pull branch locally, set `CEDAR_CORE_MOCK` env var to `true`
- Open postman, and pull in new postman collection from this branch
- Navigate to CEDAR -> Core -> Exchanges, and run the new request in there
- Confirm you receive expected mock data based on `cedarcoremock/exchange.go`



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
